### PR TITLE
Custom Logger Example Project

### PR DESCRIPTION
**Pull request by aziemp66**
**ONLY MERGE AFTER dev/project-structure is merged**

### CODE REVIEW

File: logger-example/logger/init.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+	logger = log
+	appName = name
+	defer logger.Sync()
+}
+
+func Info(fileName, methodName, msg string) {
+	logger.Info(msg,
+		zap.String("app_name", appName),
+		zap.String("trace_id", uuid.NewString()),
```
Comment (ermasafira) [inline logger-example/logger/init.go]: trace_id nya kita buat bs di-passing aja, pake `context.Context`

File: logger-example/logger/init.go [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+		zap.String("file_name", fileName),
```
Comment (ermasafira) [inline logger-example/logger/init.go]: dapetin filename filepath pake Caller aja

https://stackoverflow.com/a/38551362

File: logger-example/logger/init.go
Comment (ermasafira) [inline logger-example/logger/init.go:1]: ini ditaro di dalem folder pkg ya

File: logger-example/logger/init.go (hunk 2 above)
Comment (aziemp66) [reply logger-example/logger/init.go]: aman

**Pull request merged**